components/common/ReportMissingInfo.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Modal from "components/modal/Modal";
+import Input from "components/forms/Input";
+import Submit from "components/forms/Submit";
+
+const ReportMissingInfo = () => (
+    <Modal target="report-missing-info"
+        text="Report missing or outdated information"
+        title="Report missing or outdated information">
+        <p className="small font-italic">
+            With the help of some truly incredible pro bono partners, FreeFrom was able to complete research on policies impacting survivors’ financial security in every state and D.C. One of our priorities is to ensure that all of the information in our tool is correct and up-to-date. To that end, we welcome any correction on inaccurate or incomplete information. Thank you in advance for helping us make the Policy Map and Scorecard the best tool it can possibly be!
+        </p>
+        <form>
+            <Input name="what-is-outdated" required={ true } label="What information is missing or outdated? *
```

Comment:
What does this `*` do? Remove the new line character?

Reply:
Oh, it's a literal `*` character. We might want to remove it, VoiceOver reads this as "What information is missing or outdated? Star. Please provide as much information as you can. Star"
```suggestion
            <Input name="what-is-outdated" required={ true } label="What information is missing or outdated?
```

Reply:
Ah, gotcha. Yea, I just copy pasta'ed this from what I found in `index.html`

https://github.com/RagtagOpen/freefrom-map-frontend/blob/c76018945051f37e7a8ff7266e0f51c6c0d5a88c/pages/index.js#L19-L20

But yea, I can definitely remove!

Reply:
Removed in df12ae1081070a1769c304094638da87cbf5951f